webapp/src/reducers/index.ts

```diff
@@ -59,7 +63,8 @@ const createModalVisible = (state = false, action) => {
     }
 };
 
-const createModal = (state = '', action) => {
+// TODO: Crosscheck types
+const createModal = (state = '', action: GenericAction): any => {
```

Comment:
The default value for this reducer was mistakenly set to a string, and the function otherwise returns objects. @AGMETEOR What do you think about making the default value `{}` and this type as the return type?

```ts
type CreateModalValue = {
	postId: string;
	description: string;
	channelId: string;
} | {};
```

Reply:
@mickmister implemented your feedback on this. Thanks for the review :)